layouts/partials/footer.html

```diff
@@ -12,13 +12,15 @@
         <div class="grid-flex-cell">
           <ul class="sitemap unstyled">
             <li class="sitemap-head">IPFS</li>
-            <li><a href="//github.com/ipfs/ipfs">GitHub</a></li>
-            <li><a href="/team">Team</a></li>
             <li><a href="//docs.ipfs.io/guides/guides/install/">Install</a></li>
-            <li><a href="/media">Media</a></li>
+            <li><a href="//github.com/ipfs/ipfs">GitHub</a></li>
             <li><a href="//docs.ipfs.io/">Docs</a></li>
             <li><a href="//docs.ipfs.io/#community">Community</a></li>
             <li><a href="//discuss.ipfs.io">Forum</a></li>
+            <li><a href="https://awesome.ipfs.io/">Awesome IPFS</a></li>
```

Comment:
Similar caveat that this site hasn't been been maintained for some time, so I'm not sure we should be actively promoting it.

Reply:
True that it's being lightly maintained, but asserting that it's still useful as a source of inspiration. Requesting that we keep it in -- @cwaring, please advise.